src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -125,8 +128,20 @@ public void robotInit() {
      */
     @Override
     public void robotPeriodic() {
+        boolean precisionTurningEngaged = driverRight.getTrigger();
+        if (precisionTurningEngaged) {
+            driverLeft.setDeadband(driverLeft.getDeadband() * PRECISION_TURNING_SCALING_FACTOR);
```

Comment:
Do we want the change in deadband size to be the same as the increase in sensitivity of the turn input?